controllers/profiles/common/mutate_visitors.go

```diff
@@ -105,32 +105,28 @@ func ServiceMutateVisitor(workflow *operatorapi.SonataFlow) MutateVisitor {
 	}
 }
 
-func WorkflowPropertiesMutateVisitor(ctx context.Context, catalog discovery.ServiceCatalog,
-	workflow *operatorapi.SonataFlow, platform *operatorapi.SonataFlowPlatform) MutateVisitor {
+func UserPropertiesMutateVisitor(ctx context.Context, catalog discovery.ServiceCatalog,
```

Comment:
Maybe we should name this ManagedPropertiesMutateVisitor instead, wdyt?

Reply:
Pls see the updates on the latest commit